tutorials/polkadot-sdk/parachains/obtain-coretime.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Obtain Coretime
+description: This guide shows you how to obtain coretime for block production using on-demand or bulk options.
+---
+
+## Introduction
+
+Coretime is the mechanism in which validation resources are allocated from the relay chain to the respective [task](../../../polkadot-protocol/glossary.md#task), such as a parachain. A parachain could only produce blocks and finalize them on the relay chain by obtaining coretime.
+
+There are two ways to obtain coretime:
+
+- **Bulk coretime** - Bulk coretime allows you to obtain a core or part of a core. It is purchased for some time, up to 28 days. It must be renewed once the lease finishes
+- **On-Demand coretime** - On-demand coretime allows you to buy coretime on a block-by-block basis
+
+In this tutorial, you will:
+
+- Learn about the different coretime interfaces available
+- Learn how to purchase a core via bulk coretime
+- Assign a task / parachain to the core for block production
+- Alternatively, use on-demand coretime to produce blocks as required
+
+## Prerequisites 
+
+Before proceeding, you should have the following items:
+
+!!!note
+ You may disregard these prerequisites if you only wish to purchase a core without a parachain.
+
+- A parachain ID
+- A chain specification
+- A registered parathread with the correct genesis, runtime, and parachain ID that matches the chain specification
+- A properly configured and synced (with the relay chain) collator
+
+Once the above is complete, obtaining coretime is the last step to enable your parachain to start producing and finalizing blocks using the relay chain's validator set. If you don't, refer to the previous tutorial: [Deploy on Paseo TestNet](todo).
+
+## Purchasing Bulk Coretime
+
+Purchasing bulk coretime involves purchasing a core from the [Coretime Chain](../../../polkadot-protocol/architecture/system-chains/coretime.md), which has an instance of [`pallet_broker`](https://paritytech.github.io/polkadot-sdk/master/pallet_broker/index.html){target=\blank} (the Broker pallet). Although this can be done via sending extrinsics through a tool like Polkadot.js Apps, there are user interfaces for purchasing and managing bulk coretime:
```

Comment:
```suggestion
Purchasing bulk coretime involves purchasing a core from the [Coretime Chain](/polkadot-protocol/architecture/system-chains/coretime/){target=\_blank}, which has an instance of [`pallet_broker`](https://paritytech.github.io/polkadot-sdk/master/pallet_broker/index.html){target=\blank} (the Broker pallet). Although this can be done via sending extrinsics through a tool like Polkadot.js Apps, there are user interfaces for purchasing and managing bulk coretime:
```